### PR TITLE
Added a setup.py and made some small CLI fixes

### PR DESCRIPTION
**Pull request by sontek**

### CODE REVIEW

**Comment by sontek** [comment]
@colinmsaunders please review.  This sets us up for doing multiprocessing next:

https://github.com/sontek/monkeystud/pull/1

**Comment by colinmsaunders** [comment]
hey man, sorry for taking so long to take a look at this, but you're going to have to show me what to do -- this python foo is beyond my skills.

**Comment by sontek** [comment]
@colinmsaunders Lets go over it after devpunks, I merged the conflicts.  Need to test to make sure the conflicts got merged properly since there was a lot of work that had been done since I originally made this PR :boom:

File: monkeystud/__init__.py [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+import Queue
+import click
 from signal import signal, SIGPIPE, SIG_DFL
 signal(SIGPIPE, SIG_DFL)
 
+CHIPS_START    = 1024               # each player starts with 1024 chips
+ANTE           = 6                  # ante is 1 / 64th total chip count each
 MAX_TIME       = 100.0              # bot can be no more than 100x slower
-
-CHIPS_START    = 1000               # each player starts with 1000 chips
```
Comment (colinmsaunders) [inline monkeystud/__init__.py]: change chips_start back to 1000

File: README.rst [hunk 2]
```diff
@@ -50,29 +50,35 @@ To implement a bot, first get a copy of the game:
```
Comment (colinmsaunders) [inline README.rst]: The website expects readme to be .md. Should I change it to .rst?

**Pull request merged**